docs/io/configuration/components/supernova.rst

```diff
@@ -4,9 +4,24 @@
 Supernova Configuration
 ***********************
 
-The supernova component of the configuration file contains some key information about the supernova being modeled:
+The supernova component of the configuration file contains some key information about the supernova being modeled, namely the time since the supernova and the luminosity the user wishes TARDIS should output:
 
 .. jsonschema:: schemas/supernova.yml
 
-During a run of TARDIS, we attempt to converge the output spectrum to match the requested luminosity
-(see :ref:`est_and_conv`).
\ No newline at end of file
+As luminosity (in units of energy/s) is computed by integrating over the spectral luminosity (in units of energy/s/wavelength), TARDIS sums over all discrete energy packets to compute luminosity when ran, attempting to converge the output spectrum to match the luminosity requested (see :ref:`est_and_conv`). The range over which TARDIS sums these energy packets is set from 0 to infinity via the `luminosity_wavelength_start` and `luminosity_wavelength_end`, respectively, to generate a spectrum whose luminosity is known across the entire spectrum by default. However, if in the event only the luminosity within a certain range of wavelengths is known, then `luminosity_wavelength_start` and `luminosity_wavelength_end` can be changed as necessary to reflect this, allowing TARDIS to attempt to create a spectrum whose luminosity within the set range will converge to the value defined in `luminosity_requested`.
```

Comment:
maybe also mention that luminosity can be given in standard units (such as erg/s or J/s) or logarithmic units such as log_lsun